### DISCUSSION TITLE
Tamil (தமிழ்)

### DISCUSSION THREAD

**Discussion by Energy007ops** [sovenor/bitcoin-rocks#37] [Translations]
there is no translation in tamil

**Comment by sovenor** [comment]
Hi @Energy007ops! Is a Tamil translation something you are willing to work on?

You can check out our [Translations Guide](https://github.com/sovenor/bitcoin-rocks/blob/main/CONTRIBUTING.md#translations) to see how this process works and determine if it's something you're willing to work on. We would love to add Tamil to the website with your help!

**Comment by Energy007ops** [comment]
Yes but little bit confusion is there

**Reply by sovenor** [reply]
Do you have specific questions about the process?

**Comment by sovenor** [comment]
Tamil is now available across the entire website. Thanks to @Energy007ops for the request.

You can select Tamil in the language dropdown at the top of the page. If your browser language is set to Tamil, it should automatically choose Tamil for you when the page loads.
